### DISCUSSION TITLE
recipes/tarts/shallot-onion-and-chive-tart/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#468] [Comments]
# recipes/tarts/shallot-onion-and-chive-tart/

A collection of my recipes.

https://nicholaswilde.io/recipes/tarts/shallot-onion-and-chive-tart/

**Comment by nicholaswilde** [comment]
200 ml double cream.

**Comment by nicholaswilde** [comment]
Left over with about a 3rd of the egg and onion filling.
